### PR TITLE
composer run for launching code quality

### PR DESCRIPTION
**Pull request by roberto-butti**
## Pull request type

Jira Link: no internal

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Other (please describe): added composer scripts for launching tests and php code stantards

## How to test this PR

try to execute
```
composer run all-check
```

## What is the new behavior?

- vendor/bin/php-cs-fixer fix
- vendor/bin/pest

are executed

## Other information

### CODE REVIEW

File: .php-cs-fixer.php [hunk 1]
```diff
@@ -1,6 +1,14 @@
 <?php
 
-$finder = PhpCsFixer\Finder::create();
+$finder = PhpCsFixer\Finder::create()
+//$finder = Symfony\Component\Finder\Finder::create()
```
Comment (joaokamun) [inline .php-cs-fixer.php:4]: Can we remove this line?

File: .php-cs-fixer.php [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+    ->in([
+        __DIR__ . '/src',
+        //__DIR__ . '/tests',
```
Comment (joaokamun) [inline .php-cs-fixer.php:7]: This one can be removed too.

**Review by joaokamun — CHANGES_REQUESTED** [review]
Looks good, just made a few requests and we can merge it.

File: .php-cs-fixer.php (hunk 1 above)
Comment (roberto-butti) [reply .php-cs-fixer.php]: sure!

File: .php-cs-fixer.php (hunk 2 above)
Comment (roberto-butti) [reply .php-cs-fixer.php]: sure!

**Review by joaokamun — APPROVED** [review]
Nice work!

**Pull request merged**